bin/encodings

```diff
@@ -52,6 +52,7 @@ when "big5"         then lists("big5", 0...0x10000, Encoding::Big5)
 when "cp850"        then table(Encoding::CP850)
 when "cp852"        then table(Encoding::CP852)
 when "cp855"        then table(Encoding::CP855)
+when "cp863"       then table(Encoding::CP863)
```

Comment:
Typically we're only putting the main constants in here, and since `Encoding::CP863` is an alias of `Encoding::IBM863`, we don't need it here.